### PR TITLE
chapter 7

### PR DESCRIPTION
**Pull request by meimei0111**

### CODE REVIEW

**Comment by meimei0111** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,3 +1,35 @@
 class UsersController < ApplicationController
-  def new; end
+	before_action :load_user, only: :show
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]:
```suggestion
  before_action :load_user, only: :show
```

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-    get "/help", to: "static_pages#help"
-    get "/about", to: "static_pages#about"
-    get "/signup", to: "users#new"
-    get "users/new"
-  end 
+	root "static_pages#home"
+	get "/help", to: "static_pages#help"
+	get "/about", to: "static_pages#about"
+	get '/contact', to: 'static_pages#contact'
+	get '/signup', to: 'users#new'
```
Comment (saobv-2223) [inline config/routes.rb]: using "

**Comment by meimei0111** [comment]
ready

File: app/views/shared/_error_messages.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+	<div id="error_explanation">
```
Comment (saobv-2223) [inline app/views/shared/_error_messages.html.erb]:
```suggestion
  <div id="error_explanation">
```

**Review by saobv-2223 — COMMENTED** [review]
fix comment and merge

**Pull request merged**